tests/test-cases/Group1-Docker-Commands/1-32-Docker-Plugin.robot

```diff
@@ -40,10 +40,10 @@ Docker plugin disable
     Should Be Equal As Integers  ${rc}  1
     Should Contain  ${output}  does not yet support plugins
 
-Docker plugin inspect
-    ${rc}  ${output}=  Run And Return Rc And Output  docker1.13 %{VCH-PARAMS} plugin inspect test-plugin
-    Should Be Equal As Integers  ${rc}  1
-    Should Contain  ${output}  does not yet support plugins
+#Docker plugin inspect
```

Comment:
Instead of commenting this test altogether, we should have something like

```
Docker plugin inspect

    Pass execution  Test not implemented till plugin support is added
```

See https://github.com/vmware/vic/blob/master/tests/test-cases/Group6-VIC-Machine/6-13-TLS.robot#L54